lib/matplotlib/ticker.py

```diff
@@ -1146,10 +1146,10 @@ def __init__(
         Parameters
         ----------
         use_overline : bool, default: False
-            If x > 1/2, with x = 1-v, indicate if x should be displayed as
-            $\overline{v}$. The default is to display $1-v$.
+            If x > 1/2, with x = 1 - v, indicate if x should be displayed as
+            $\overline{v}$. The default is to display $1 - v$.
 
-        one_half : str, default: r"\frac{1}{2}"
+        one_half : str, default: r"\\frac{1}{2}"
```

Comment:
The backslash was missing from the rendered version, so will have to check it this gets it back...

Reply:
Maybe make the whole docstring raw. Then you don't need backslash doubling here (but can keep the r prefix for copying).

Reply:
It is raw, which is why I had to test to see what happened. I do not really understand why I need to escape it, but that seems to do the trick...

https://github.com/matplotlib/matplotlib/blob/a254b687df97cda8c6affa37a1dfcf213f8e6c3a/lib/matplotlib/ticker.py#L1145-L1153

Reply:
Ok, I'm confused. This feels like a bug in sphinx. But I question that it's worth tracking down.

Reply:
Yeah, not obvious why it is required.